image_data_guide/01_download_data.ipynb

```diff
@@ -0,0 +1,546 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Downloading and Combining Datasets (Part 1/4)\n",
+    "**Download** | Structure | Preprocess | Train"
+   ]
+  },
+  {
```

Comment:
In the other notebooks you have a table of contents at the top. For consistency, let's add on here. Brownie points if they can link within the page.

Reply:
Added an overview section to match the other notebooks